API.md

```diff
@@ -106,6 +118,8 @@ Returns a single election with some more data than the list:
 
 ## Voters API
 
+### `POST /api/mailelection/:electionId/mailinglist`
```

Comment:
Minor nit: we're using `:electionId` and `<election_id>` to mean the same thing.